tensilelite/Tensile/Tests/common/gemm/large_size.yaml

```diff
@@ -0,0 +1,60 @@
+TestParameters:
+  marks: [skip-gfx1100, skip-gfx1101, skip-gfx1102] # not supported yet
+
+GlobalParameters:
+  MinimumRequiredVersion: 4.14.0
+  SleepPercent: 50
+  NumElementsToValidate: 128
+  DataInitTypeBeta: 0
+  DataInitTypeAlpha: 1
+  NewClient: 2
+  CSVExportWinner: 1
+  CSVMergeSameProblemID: 1
+  Device: 0
+
```

Comment:
Should you add rotating buffer setting here for testing?

Reply:
yes. fixed.